tests/integration/cheatcodes/deploy_contract/deploy_contract_test.cairo

```diff
@@ -256,3 +256,25 @@ func test_constructor_no_args_executed{syscall_ptr: felt*, range_check_ptr}() {
 
     return ();
 }
+
+@contract_interface
+namespace EventEmitterContainer {
+    func emit() {
+    }
+}
+
+@event
+func fake_event() {
+}
+
+
+@external
+func test_emitting_events_from_user_contract_constructor_and_from_current_contract{syscall_ptr: felt*, range_check_ptr}() {
+    // https://github.com/software-mansion/protostar/issues/823
```

Comment:
FYI that's how I'd write this, but this is fine and there's no need to change :)

```suggestion
// https://github.com/software-mansion/protostar/issues/823
@external
func test_issue_823{syscall_ptr: felt*, range_check_ptr}() {
```

Reply:
I'd rather keep a descriptive name for a test case. The link could be dead in the future.